webapp/channels/src/components/suggestion/at_mention_provider/at_mention_suggestion.tsx

```diff
@@ -190,15 +199,20 @@ const AtMentionSuggestion = React.forwardRef<HTMLLIElement, SuggestionProps<Item
         <SuggestionContainer
             ref={ref}
             {...props}
+            aria-labelledby={ids.atMention}
+            aria-describedby={joinIds(ids.description, ids.youElement, ids.status, ids.botTag, ids.sharedIcon, ids.guestTag, ids.groupMembers)}
```

Comment:
The label for at-mention suggestions is `@username`/`@group`, and the description will include any of these that that exist:
- The text description for `@all`/`@channel`/`@here`
- The "(you)" text for when mentioning yourself
- The user's status
- If the user is a bot
- If the user or group is from a remote server in a shared channel
- If the user is a guest
- If this is a group, the number of users in that group

When we give `aria-describedby` a list of IDs, it constructs the description out of any elements that exist with those IDs, so we can just pass all of the IDs to it without worrying about which of these elements actually exist